js/interface/add-markers.js

```diff
@@ -233,7 +233,9 @@ Fliplet.InteractiveMap.component('add-markers', {
 
       Fliplet.Studio.emit('widget-mode', 'normal')
       this.$nextTick(() => {
-        this.initDataSourceProvider(this.dataSourceId)
+        if (!this.dataSourceProvider) {
+          return this.initDataSourceProvider(this.dataSourceId)
```

Comment:
I don't think `return` is needed here.

```suggestion
         this.initDataSourceProvider(this.dataSourceId)
```

Reply:
Done